plugins/sitoa/renderer/IprShader.cpp

```diff
@@ -309,6 +309,9 @@ void UpdatePassShaderStack(const Pass &in_pass, double in_frame)
          Shader outputShader = GetConnectedShader(passParam);
          if (outputShader.IsValid())
          {
+            // skip 'operator' dummy shader
+            if (outputShader.GetName() == L"operator")
```

Comment:
Are you sure this isn't just getting the name of that shader node instance. That is if you copy it, it will be named "operator1"?
Could you try that just to make sure it works?
If your code dosn't work I recommend you use the code that is used everywhere else in SItoA:
```suggestion
            if (GetShaderNameFromProgId(outputShader.GetProgID()) == L"operator")
```

Reply:
Thanks for catching that, I will switch to the GetShaderNameFromProgId.